packages/react-native/src/index.ts

```diff
@@ -3,3 +3,4 @@ export {
   InAppMessagingProvider,
   useInAppMessaging,
 } from '@aws-amplify/ui-react-core';
+export * from './primitives';
```

Comment:
We cannot export these directly as that exposes them publicly, but you should be able to import the primitives from 
`@aws-amplify/ui-react-native/dist/primitives` in the example app

Reply:
Ack